launcher/minecraft/ComponentUpdateTask.h

```diff
@@ -25,12 +25,12 @@ class ComponentUpdateTask : public Task
     void executeTask();
 
 private:
-    void loadComponents();
-    void resolveDependencies(bool checkOnly);
+    void loadComponents(bool firstRun);
+    void resolveDependencies(bool firstRun, bool checkOnly);
```

Comment:
```suggestion
    void resolveDependencies(bool checkOnly, bool firstRun);
```

all the other methods have this at the end, so this should probably too :>